### DISCUSSION TITLE
Even though there are 3 nameservers under my /etc/resolv.conf file, it gives an error saying kubelet limit is exceeded. Can you help?

### DISCUSSION THREAD

**Discussion by cengizhanR** [rancher/rke2#6823] [General]
Provider: RKE2
Kubernetes Version: v1.30.4 +rke2r1
Architecture: Amd64
OS: Red Hat Enterprise Linux 9.4 (Plow)
Container Runtime: containerd://1.7.20-k3s1

```
[root@SBLG-UISRV01 logs]# cat /etc/resolv.conf
# Generated by NetworkManager
```

```

```

**Comment by brandond** [comment]
This is coming from core Kubernetes, not anything specific to RKE2:
https://github.com/kubernetes/kubernetes/blob/c970a46bc1bcc100bbbfabd5c12bd4c5d87f8aea/pkg/apis/core/validation/validation.go#L2944-L2953

**Reply by cengizhanR** [reply]
Thank you very much.
